### PR TITLE
Fix variable wrapper

### PR DESCRIPTION
**Pull request by yanndupis**
Hey, 

When sending a tf.Variable, it was returning at tf.Tensor pointer instead of a tf.Variable pointer. Also had to add ResourceVariable support

### CODE REVIEW

File: syft_tensorflow/hook/hook.py [hunk 1]
```diff
@@ -284,7 +284,7 @@ def _add_methods_from_native_tensor(tensor_type: type, syft_type: type):
 
     @classmethod
     def create_wrapper(cls, child_to_wrap, *args, **kwargs):
-        if isinstance(child_to_wrap.object_type, tf.Variable):
+        if type(child_to_wrap.object_type)==type(tf.Variable):
```
Comment (bendecoste) [inline syft_tensorflow/hook/hook.py]: Why this change?

Comment (jvmancuso) [inline syft_tensorflow/hook/hook.py]: why the change? type(x) == type(y) is discouraged by PEP

Comment (yanndupis) [reply syft_tensorflow/hook/hook.py]: it's because `child_to_wrap.object_type` is not instantiated so it's like doing `isinstance(tf.Variable, tf.Variable)` and it will return False. 

child_to_wrap.object_type is defined [here](https://github.com/OpenMined/PySyft-TensorFlow/blob/c8cee90667eb8ea41efda46faf25307411f22537/syft_tensorflow/tensor/variable.py#L236)

do you have another solution?

Comment (yanndupis) [reply syft_tensorflow/hook/hook.py]: see replied to Ben

Comment (bendecoste) [reply syft_tensorflow/hook/hook.py]: In that case the type wrapper is unnecessary, you can just do `if child_to_wrap.object_type == tf.Variable`

Comment (yanndupis) [reply syft_tensorflow/hook/hook.py]: fixed, let's forget about this question haha. thanks
